### DISCUSSION TITLE
[Feature Request] Add endContent to AccorionItem

### DISCUSSION THREAD

**Discussion by 7prog** [heroui-inc/heroui#4691] [Feature requests]
### Is your feature request related to a problem? Please describe.

Add an end content in addition to start content to the accordion item for buttons and actions

### Describe the solution you'd like

both start and end content for the accordion item

### Describe alternatives you've considered

I've used both the title and start content to add actions to the end of the trigger

**Comment by Jlassiezzeddine** [comment]
I think this is useful to have both startContent and endContent on the accordion component,

I'm working on a PR

**Comment by sescotti** [comment]
hi @Jlassiezzeddine, do you remember what happened with this in the end? saw a bunch of PRs that failed some checks and no further updates. did you find a workaround?

**Comment by criskell** [comment]
What is the state of this issue?
